src/zjs_arduino101_pins.json

```diff
@@ -0,0 +1,7 @@
+{
+    "module":"arduino101_pins",
+    "require":"arduino101_pins",
+    "targets":["arduino_101"],
+    "zjs_config": ["-DBUILD_MODULE_A101"],
```

Comment:
Try to be consistent with a space after `:`, hopefully some judicious sed could make the work easy.